src/widgets/health-monitoring/HealthMonitoringCpu.tsx

```diff
@@ -0,0 +1,118 @@
+  const { t } = useTranslation('modules/health-monitoring');
+  const toFahrenheit = (value: number) => {
+    return Math.round(value * 1.8 + 32);
+  };
+
+  interface LoadDataItem {
+    label: string;
+    stats: number;
+    progress: number;
+    color: string;
+  }
+
+  const loadData = [
+    {
+      label: `${t('info.load')} (1min)`,
+      stats: info.loadAverage['1min'],
+      progress: info.loadAverage['1min'],
+      color: 'teal',
+    },
+    {
+      label: `${t('info.load')} (5min)`,
+      stats: info.loadAverage['5min'],
+      progress: info.loadAverage['5min'],
+      color: 'blue',
+    },
+    {
+      label: `${t('info.load')} (15min)`,
+      stats: info.loadAverage['15min'],
+      progress: info.loadAverage['15min'],
+      color: 'red',
+    },
+  ] as const;
+
+  return (
+    <Group position="center">
+      <RingProgress
+        roundCaps
+        size={140}
+        thickness={12}
+        label={
+          <Center style={{ flexDirection: 'column' }}>
+            {info.cpuUtilization.toFixed(2)}%
+            <HoverCard width={280} shadow="md" position="top">
+              <HoverCard.Target>
+                <IconCpu size={40} />
+              </HoverCard.Target>
+              <HoverCard.Dropdown>
+                <Text fz="lg" tt="uppercase" fw={700} c="dimmed" align="center">
+                  {t('info.load')}
+                </Text>
+                <Flex
+                  direction={{ base: 'column', sm: 'row' }}
+                  gap={{ base: 'sm', sm: 'lg' }}
+                  justify={{ sm: 'center' }}
+                >
+                  {loadData.map((load: LoadDataItem) => (
+                    <RingProgress
+                      size={80}
+                      roundCaps
+                      thickness={8}
+                      label={
+                        <Text color={load.color} weight={700} align="center" size="xl">
+                          {load.progress}
+                        </Text>
+                      }
+                      sections={[{ value: load.progress, color: load.color, tooltip: load.label }]}
+                    />
+                  ))}
+                </Flex>
+              </HoverCard.Dropdown>
+            </HoverCard>
+          </Center>
+        }
+        sections={[
+          {
+            value: info.cpuUtilization.toFixed(2),
+            color: info.cpuUtilization.toFixed(2) > 70 ? 'red' : 'green',
+          },
+        ]}
+      />
+      <RingProgress
+        roundCaps
+        size={140}
+        thickness={12}
+        label={
+          <Center
+            style={{
+              flexDirection: 'column',
+            }}
+          >
+            {fahrenheit ? `${toFahrenheit(cpuTemp.cputemp)}°F` : `${cpuTemp.cputemp}°C`}
+            <IconCpu size={40} />
+          </Center>
+        }
+        sections={[
+          {
+            value: cpuTemp.cputemp,
+            color:
+              cpuTemp.cputemp < 35
```

Comment:
bad check. 35° will be red, so will 60°.
Only do checks under a certain temp.

Reply:
Also, it's not the best looking thing, maybe if you could extract it to a real if statement.